server/services/employers.js

```diff
@@ -45,7 +45,14 @@ const updateSite = async (id, site) => {
 };
 
 const getSites = async () => dbClient.db[collections.EMPLOYER_SITES].findDoc({});
-const getSiteByID = async (id) => dbClient.db[collections.EMPLOYER_SITES].findDoc({ id });
+const getSiteByID = async (id) => {
+  const site = await dbClient.db[collections.EMPLOYER_SITES].findDoc({ id });
+  const hcapHires = await dbClient.db[collections.PARTICIPANTS_STATUS].count({ 'data.site': site[0].siteId, 'data.nonHcapOpportunity': 'false' });
+  const nonHcapHires = await dbClient.db[collections.PARTICIPANTS_STATUS].count({ 'data.site': site[0].siteId, 'data.nonHcapOpportunity': 'true' });
+  site[0].hcapHires = hcapHires;
+  site[0].nonHcapHires = nonHcapHires;
+  return site;
```

Comment:
It's kinda weird behavior to return an array where we expect a single site. Everywhere `getSiteByID` is used, the first value is pulled from the returned array.

Reply:
Yeah if we're returning a single object elsewhere, makes sense to me for consistency - I can tackle this change while Zenen is heads-down on the other ticket

Reply:
Actually I'm not sure what's going on here. The query is the exact same for sites as it is for other single entities like a single employer, and the caller (server.js) doesn't handle it like it's an array at all.

I'm pretty sure this is exactly the point I was confused by the last time I was working in this area of the code as well - the site ID wasn't being logged properly, I concluded from running tests locally that the site was coming back as an array and accessed the ID via the first entry, but then when I pushed that code it turned out not to be correct 🤷‍♀️ 

Check where getSiteByID is called in server.js (/employer-sites/:id)